uaclient/messages.py

```diff
@@ -785,9 +785,9 @@ class TxtColor:
     "livepatch-not-enabled", "canonical-livepatch snap is not installed."
 )
 
-LIVEPATCH_ERROR_INSTALL_ON_CONTAINER = NamedMessage(
+SERVICE_ERROR_INSTALL_ON_CONTAINER = FormattedNamedMessage(
     "livepatch-error-install-on-container",
-    "Cannot install Livepatch on a container.",
+    "Cannot install {title} on a container.",
 )
```

Comment:
The name is still `"livepatch-error-install-on-container"` - should that change too?